sentry-android/src/main/java/io/sentry/android/AndroidSentryClientFactory.java

```diff
@@ -61,8 +63,10 @@ public AndroidSentryClientFactory(Application app) {
      * Construct an AndroidSentryClientFactory using the specified Android Context.
      *
      * @param ctx Android Context.
+     * @param lookup the lookup for locating the configuration
      */
-    public AndroidSentryClientFactory(Context ctx) {
+    public AndroidSentryClientFactory(Context ctx, Lookup lookup) {
```

Comment:
This would be a breaking change then I guess? We don't offer an overload that doesn't take `Lookup` where we instantiate it ourselves to be backwards compatible?